lib/daru/maths/statistics/dataframe.rb

```diff
@@ -6,6 +6,27 @@ module DataFrame
         def mean
           compute_stats :mean
         end
+        
+        # Calculate variance of numeric vectors
+        def variance_sample
+          compute_stats :variance_sample
```

Comment:
All these methods can be replaced with `define_method`. You can add documentation with the YARD `@method` syntax.